src/tasks/DockerRunTaskDefinitionBase.ts

```diff
@@ -21,7 +21,7 @@ export interface DockerContainerPort {
 export interface DockerContainerVolume {
     localPath: string;
     containerPath: string;
-    permissions?: 'ro' | 'rw';
+    permissions?: 'ro' | 'rw' | 'ro,z' | 'rw,z';
```

Comment:
Should we add the `ro,z` and `rw,z` options to the [enum](https://github.com/microsoft/vscode-docker/blob/main/package.json#L1212-L1215) in `package.json` and maybe also the [description string](https://github.com/microsoft/vscode-docker/blob/main/package.nls.json#L85) in `package.nls.json`? This would enable end users to use these options without the warning squiggles.

Reply:
I have added this. ptal.

Reply:
It's in `package.json` now but got removed from `DockerRunTaskDefinitionBase.ts`, was that intentional? It would need to be in both.

Reply:
Oh, also the changes to `NetCoreTaskHelper.ts` with the `,z` label on each volume mapping got removed.